pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -76,6 +76,9 @@
   @Option(name = "-retry", metaVar = "<int>", usage = "Number of retries if encountered any segment creation failure, default is 0.")
   private int _retry = 0;
 
+  @Option(name = "-failOnEmptySegment", usage = "Option to fail the segment creation if output is an empty segment.")
```

Comment:
we dont use this Command anymore right? We use the IngestionJobLauncher. You'll have to add it to one of the specs in the batchJobSpec, and also all the implementations of IngestionJobRunner

Reply:
Thanks for the pointer. Added in the latest push.